src/components/page-theme/Theme.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { ThemeProvider } from '../../context/ThemeContext';
+import CharacterPage from '../../pages/CharacterPage';
+import Header from '../header/Header';
+import './Theme.css';
+
+const Theme = () => {
+  return (
+    <ThemeProvider>
```

Comment:
The best place to wrap things in your provider is in index.js. Just wrap the whole app component